conf/messages.cy

```diff
@@ -390,24 +391,24 @@ settlorIndividualCountryOfResidency.error.required = Nodwch wlad
-settlorIndividualCountryOfResidencyPastTense.error.length = The country of residence must be 100 characters or less
-settlorIndividualCountryOfResidencyPastTense.error.invalidCharacters = The country of residence must only include letters a to z, apostrophes, commas, full stops, hyphens, round brackets and spaces
+settlorIndividualCountryOfResidencyPastTense.title = A ydych yn gwybod gwlad breswyl hysbys ddiwethaf y setlwr?
+settlorIndividualCountryOfResidencyPastTense.heading = A ydych yn gwybod beth oedd gwlad breswyl hysbys ddiwethaf {0}’s?
+settlorIndividualCountryOfResidencyPastTense.checkYourAnswersLabel = A ydych yn gwybod beth oedd gwlad breswyl hysbys ddiwethaf {0}’s?
+settlorIndividualCountryOfResidencyPastTense.error.required = Nodwch wlad
+settlorIndividualCountryOfResidencyPastTense.error.length = Mae’n rhaid i’r wlad breswyl fod yn 100 o gymeriadau neu lai
+settlorIndividualCountryOfResidencyPastTense.error.invalidCharacters = Mae’n rhaid i’r wlad breswyl gynnwys y llythrennau a i z yn unig
 
 settlorIndividualCountryOfResidencyYesNo.checkYourAnswersLabel = A ydych yn gwybod gwlad breswyl {0}?
 settlorIndividualCountryOfResidencyYesNo.error.required = Dewiswch ‘Iawn’ os ydych yn gwybod gwlad breswyl y setlwr
 settlorIndividualCountryOfResidencyYesNo.heading = A ydych yn gwybod gwlad breswyl {0}?
 settlorIndividualCountryOfResidencyYesNo.hint = Fel arfer, y wlad breswyl yw lle mae’r setlwr yn byw ac yn gweithio’r rhan fwyaf o’r amser yn ystod y flwyddyn dreth.
 settlorIndividualCountryOfResidencyYesNo.title = A ydych yn gwybod gwlad breswyl y setlwr?
 
-settlorIndividualCountryOfResidencyYesNoPastTense.title = Do you know the settlor’s last known country of residence?
-settlorIndividualCountryOfResidencyYesNoPastTense.heading = Do you know {0}’s last known country of residence?
-settlorIndividualCountryOfResidencyYesNoPastTense.checkYourAnswersLabel = Do you know {0}’s last known country of residence?
-settlorIndividualCountryOfResidencyYesNoPastTense.hint = The last known country of residence is usually where the settlor lived and worked most of the time during the tax year.
-settlorIndividualCountryOfResidencyYesNoPastTense.error.required = Select yes if you know the settlor’s last known country of residence
+settlorIndividualCountryOfResidencyYesNoPastTense.title = A ydych yn gwybod gwlad breswyl hysbys ddiwethaf y setlwr?
+settlorIndividualCountryOfResidencyYesNoPastTense.heading = A ydych yn gwybod beth oedd gwlad breswyl hysbys ddiwethaf {0}’s?
+settlorIndividualCountryOfResidencyYesNoPastTense.checkYourAnswersLabel = A ydych yn gwybod beth oedd gwlad breswyl hysbys ddiwethaf {0}’s?
+settlorIndividualCountryOfResidencyYesNoPastTense.hint = Fel arfer, y wlad breswyl yw lle mae’r setlwr yn byw ac yn gweithio’r rhan fwyaf o’r amser yn ystod y flwyddyn dreth.
+settlorIndividualCountryOfResidencyYesNoPastTense.error.required = Dewiswch ‘Iawn’ os ydych yn gwybod gwlad breswyl y setlwr
```

Comment:
`settlorIndividualCountryOfResidencyYesNoPastTense.error.required` doesn't match the translation in the document